app/views/idv/cancellations/destroy.html.erb

```diff
@@ -1,24 +1,24 @@
-<% title t('headings.cancellations.confirmation') %>
-
-<%= image_tag(asset_url('alert/fail-x.svg'), alt: '', width: 54) %>
-
-<h1 class="margin-top-4 margin-bottom-2">
-  <%= t('headings.cancellations.confirmation') %>
-</h1>
-
-<ul class='list-reset red-dots'>
-  <li><%= t('idv.cancel.warnings.warning_2') %></li>
-  <li><%= t('idv.cancel.warnings.warning_3', app: APP_NAME) %></li>
-  <li><%= t('idv.cancel.warnings.warning_4') %></li>
-  <li><%= t('idv.cancel.warnings.warning_5') %></li>
-</ul>
-
-<% if decorated_session.sp_name %>
-  <div class='margin-top-2'>
-    <%= link_to "‹ #{t('links.back_to_sp', sp: decorated_session.sp_name)}", @return_to_sp_path %>
-  </div>
-<% else %>
-  <div class='margin-top-2'>
-    <%= link_to "‹ #{t('links.back_to_sp', sp: t('links.my_account'))}", account_url %>
-  </div>
-<% end %>
+<%= render(
+      'idv/shared/error',
+      heading: @hybrid_session ?
+        t('idv.cancel.headings.confirmation.hybrid') :
+        t('headings.cancellations.confirmation'),
+      options: [
+        !@hybrid_session && decorated_session.sp_name && {
+          text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_session.sp_name),
+          url: @return_to_sp_path,
+        },
```

Comment:
This might be a change we don't want to make, but in standardizing this desktop cancellation confirmation screen to use the same error layout we use elsewhere, I had opted to revise the "Back to SP" link to be similar to how we show it as a troubleshooting option. In retrospect, I could see this being a change we don't necessarily want to make. If we don't, I can make this part of the content instead, and not show troubleshooting list.

cc @anniehirshman-gsa 

Before|After
---|---
![Screen Shot 2021-05-28 at 12 54 24 PM](https://user-images.githubusercontent.com/1779930/120017730-35a5aa00-bfb4-11eb-858e-9952202e2e27.png)|![Screen Shot 2021-05-28 at 12 55 01 PM](https://user-images.githubusercontent.com/1779930/120017745-39d1c780-bfb4-11eb-8309-51b10d918737.png)

Reply:
I opted to restore this to the previous behavior in f3afbf16a, rather than deal with this here, since it's not really something we intended to change.

![image](https://user-images.githubusercontent.com/1779930/120028985-7822b300-bfc3-11eb-91cd-580c86ec4459.png)